VikingQuiz.Api/Repositories/GameRepo.cs

```diff
@@ -53,6 +54,27 @@ public Game GetGameById(int id)
         {
             return ctx.Game.Find(id);
         }
+
+        public Game GetGameByCode(string code)
+        {
+            Game foundGame = ctx.Game.Where(x => x.Code == code)
+                .Select(x => new Game { Id = x.Id, QuizId = x.QuizId, GameDate = x.GameDate, Code = x.Code })
+                .FirstOrDefault();
+
+            return foundGame;
+        }
+
+        public string GenerateCode()
+        {
+            const string chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+            string code = "";
+            for(int i = 0; i < 6; i++)
```

Comment:
So many magic numbers here in this method. We should use the length of chars string instead of 35 and some constant instead of the number 6